### PR TITLE
Added west virginia court website test

### PR DESCRIPTION
**Pull request by bahasadyr**

### CODE REVIEW

File: src/test/java/BahaSadTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        getDriver().get("http://www.courtswv.gov/");
+        getDriver().manage().window().maximize();
+        String title = getDriver().getTitle();
+        String expectedTitle = "West Virginia Judiciary - Supreme Court of Appeals of West Virginia";
+        Assert.assertEquals(title, expectedTitle);
+        WebElement searchBar = getDriver().findElement(By.id("cludo-search-content-form-input"));
+        WebElement searchButoon = getDriver().findElement(By.xpath("//button[contains(text(),'Search')]"));
+        searchBar.sendKeys("Legal");
+        searchButoon.click();
+        String expetitle = "West Virginia Judiciary";
```
Comment (Belarus8500) [inline src/test/java/BahaSadTest.java:21]: camel case

**Review by Belarus8500 — APPROVED** [review]
all Asserts  are better to keep at the end of the test and separate with an empty line, like the last one

File: src/test/java/BahaSadTest.java (hunk 1 above)
Comment (bahasadyr) [reply src/test/java/BahaSadTest.java]: changed camelCase, please review

**Review by Belarus8500 — COMMENTED** [review]
other then that looks good

File: src/test/java/BahaSadTest.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    @Test
+    public void testWestVirginiaCourt() throws InterruptedException {
+
+        getDriver().get("http://www.courtswv.gov/");
+        getDriver().manage().window().maximize();
+        String title = getDriver().getTitle();
+        String expectedTitle = "West Virginia Judiciary - Supreme Court of Appeals of West Virginia";
+        Assert.assertEquals(title, expectedTitle);
+        WebElement searchBar = getDriver().findElement(By.id("cludo-search-content-form-input"));
+        WebElement searchButoon = getDriver().findElement(By.xpath("//button[contains(text(),'Search')]"));
```
Comment (TatianaTomilina) [inline src/test/java/BahaSadTest.java:18]: looks like a typo here: searchButton

**Review by TatianaTomilina — APPROVED** [review]
other then that looks good

**Review by ke-de-ve — APPROVED** [review]

File: src/test/java/BahaSadTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        searchBar.sendKeys("Legal");
+        searchButoon.click();
+        String expectTitle = "West Virginia Judiciary";
+        String actTitle = getDriver().getTitle();
+        WebElement viewPDFFiles = getDriver().findElement(By.xpath("//a[@href='viewing-pdf-files.html']"));
+
+        Assert.assertTrue(viewPDFFiles.isDisplayed());
+
+        Assert.assertEquals(actTitle, expectTitle);
+        getDriver().quit();
```
Comment (VitalyVZH) [inline src/test/java/BahaSadTest.java:28]: тоже не нужно, драйвер закрывается в BaseTest

**Review by VitalyVZH — COMMENTED** [review]
Добавил комментарии

**Pull request merged**